samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -59,6 +61,16 @@ class BrowserFragment : BaseBrowserFragment(), BackHandler {
             view = layout
         )
 
+        webChannelFeature.set(
```

Comment:
@grigoryk should we work on getting an instance of FxA or FxA Manager passed in here?